test/e2e_node/cpumanager_test.go

```diff
@@ -813,10 +1088,8 @@ var _ = SIGDescribe("CPU Manager", ginkgo.Ordered, ginkgo.ContinueOnFailure, fra
 				for _, cnt := range pod.Spec.Containers {
 					ginkgo.By(fmt.Sprintf("validating the container %s on pod %s", cnt.Name, pod.Name))
 
-					// expect allocated CPUs to be able to fit on uncore cache ID equal to 0
-					expUncoreCPUSet, err := uncoreCPUSetFromSysFS(0)
-					framework.ExpectNoError(err, "cannot determine shared cpus for uncore cache on node")
-					gomega.Expect(pod).To(HaveContainerCPUsASubsetOf(cnt.Name, expUncoreCPUSet))
+					gomega.Expect(pod).To(HaveContainerCPUsWithSameUncoreCacheID(cnt.Name))
+					gomega.Expect(pod).ToNot(HaveContainerCPUsShareUncoreCacheWith(cnt.Name, reservedCPUs))
```

Comment:
I think we can remove this very line, which is causing the failure. We don't guarantee this behavior (or try hard to ensure).
I think my previous comment was misplaced indeed. This specific test should not check that the allocated uncore is shared, we should do only if `strict-cpu-reservation` is set. And we do that also for `full-pcpus-only` (lines 1376 and 1465) and this works as expected.